salt/_modules/metalk8s_volumes.py

```diff
@@ -132,6 +134,42 @@ def format(name):
     _get_volume(name).format()
 
 
+def is_cleaned_up(name):
+    """Check if the backing storage device for the given volume is cleaned up.
+
+    Args:
+        name (str): volume name
+
+    Returns:
+        bool: True if the backing storage device is cleaned up, otherwise False
+
+    CLI Example:
+
+    .. code-block:: bash
+
+        salt '*' metalk8s_volumes.is_cleaned_up example-volume
```

Comment:
You won't want to run this on `*`, no? A volume sits on a particular minion/host.

Reply:
Indeed. Done